1-mandatory/2-people-i-know.js

```diff
@@ -402,6 +451,16 @@ This time, I only want the full names of the people who can multitask
 */
 
 let colleaguesWhoCanMultitask = [];
+// Very similar to the previous one, we just go one
+// more layer deeper into the friend array.
+
+friends.forEach((eachFriend) => {
+  eachFriend.colleagues.forEach((eachPerson) => {
+    if (eachPerson.skills.includes("Multi-tasking")) {
+      colleaguesWhoCanMultitask.push(eachPerson.name);
+    }
+  });
+});
```

Comment:
This works perfectly well and is pretty clear!

An alternative method might be this:

```js
const colleaguesWhoCanMultitask = friends
  .map((friend) =>
    friend.colleagues
      .filter((colleague) => colleague.skills.includes("Multi-tasking"))
      .map((colleague) => colleague.name)
  )
  .flat();
```

Reply:
I added this to the file, Thankss Douglas